.travis.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           - ./cc-test-reporter after-build -t coverage.py --exit-code $TRAVIS_TEST_RESULT tests/coverage.xml
 
     - stage: delivery
-      if: type != pull_request
+      if: type != pull_request AND branch = master
```

Comment:
I wouldn't add the master branch restriction here. Here why:

Right now during the development of the package, every time we push new changes to our own fork, this travis job updates the package in the defined `OBS_PROJECT`. In my case (and some other people too), the project in my fork is enabled in travis too (with my OBS home projects in `OBS_PROJECT`). This makes the testing of the packages really easy, as I just need to commit changes to have them packaged as RPM. Really useful.

With this restriction, this delivery won't be executed more for my development branches.
And it's not harmful, as the main delivery only will be performed when changes reach `master`, that only happens when the PR is merged

Reply:
Hmm, the problem is that this will trigger OBS updates even as soon as other branches are created in the main upstream, while your use case is only for fork branches.

The way I've approached this problem is that I on the fork I simulate the exact same event that happens in the upstream, i.e. the feature branch must be merged in `master`.

If we leave out this restriction, the automation becomes prone to a number of human errors potentially causing undesired builds in OBS which would be quite painful to correct.

IMHO this kind of automation, interacting with user-facing downstream services, should be as strict as possible.

That said, if you insist I will leave this change out on this repo. We'll have do document this property though, so that future mantainers are aware that pushing branches to the main remote will trigger OBS commits (which I wouldn't expect).

Reply:
Ok, I see your point. What if we reach an intermediate solution. Something like:
```
if: type != pull_request AND (branch = master or env(ALLOW_BRANCHES))
```
Where `ALLOW_BRANCHES` is an additional environment variable that by default is `false` but in our fork travis configurations we can set to `true` to continue having the current features.
I know that it still adds some potential errors (someone changing the travis variables), but well, this can happen with the already existing variables too.

Reply:
Sounds good, I like it! This way one has to willingly do something to enable OBS builds explicitly outside the branch master.

Reply:
Ok, so let's go with that. Hopefully, `env(ALLOW_BRANCES)` will return `false` if the parameter is not set. It should be managed as `null`. So, we shouldn't even need to change the upstream travis job.
Just add an additional doc line explaining this param and that's it.

Reply:
@arbulu89 done! I called it `DELIVER_BRANCHES` and it's documented in CONTRIBUTING.md.